app/services/generate_training_data.rb

```diff
@@ -9,7 +9,7 @@ def initialize(text)
   def call
     response = @client.chat(
       parameters: {
-        model: "gpt-4",
+        model: "gpt-4-mini",
```

Comment:
Should this match the other files?

```suggestion
        model: "gpt-4o-mini",
```

Reply:
@clarissalimab good catch! it should be `gpt-4o-mini`